main.go

```diff
@@ -153,7 +185,7 @@ func main() {
 		managerOptions.HealthProbeBindAddress = "127.0.0.1:8181"
 	}
 
-	webhookOptions := setupWebhook(ctx, cfg, log, slices.Contains(crds, mongoDBMultiClusterCRDPlural), currentNamespace)
+	webhookOptions := setupWebhook(ctxWithSpan, cfg, log, slices.Contains(crds, mongoDBMultiClusterCRDPlural), currentNamespace)
```

Comment:
we don't need ctxWithSpan, IUUC ctxWithSpan is just a context but with some additional values